CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,73 @@
+# Correlation Context HTTP Header Format
```

Comment:
fyi @louiscryan this is that thing which can carry extra stuff (separate from the trace-context)

Reply:
Thanks, will change the link